env.local

```diff
@@ -6,6 +6,9 @@ DEPLOYMENT_ENV=canary
 ALCHEMY_API_KEY=
 SATSUMA_API_KEY=
 
+ETHEREUM_RPC_URL=
+SEPOLIA_RPC_URL=
+
```

Comment:
Thanks for the suggestion, but this actually called my attention to the fact that backend already has its structure for rpcURLs using `DRPC_API_KEY` - I'll update that here